[tapocontrol] new communication protocol integration / code revision (#15725)
openhab/openhab-addons: bundles/org.openhab.binding.tapocontrol/README.md

| Parameter              | Description                                                                                         |
|------------------------|-----------------------------------------------------------------------------------------------------|
| username               | Username (eMail) of your Tapo-Cloud                                                                 |
| password               | Password of your Tapo-Cloud                                                                         |
| cloudDiscovery         | Use Cloud Discovery-Service to get all in Tapo-App registered devices. Includes DeviceName. IP-Address and Encryption has to set manually                     |
| udpDiscovery           | Use UDP Discovery-Service to discover online devices in the local network. Includes Encryption and IP-Address. Results will be merged with cloud discovery    |
| onlyLocalOnlineDevices | [advanced] Uses Cloud and UPD-Discovery to get more informations but will only discover online devices via UDP |
| broadcastAddress       | [advanced] Set broadcast address to your local subnet  if you have problems with default address   |

> | discoveryInterval      | [advanced] Interval in minutes when a background device scan should be executed. Default is 60     |

Reviewer: Minor: can you fix the table markdown?

Author: Tried to fix it, but because of long text it's not perfect. if you have a better idea please post it :)

> ## Thing Configurations

Reviewer: ```suggestion
## Thing Configuration
```

> WiFi-Things needs to be configured with `ipAddress`.

Reviewer: ```suggestion
WiFi-based things needs to be configured with `ipAddress`.
```

> RF-Things need a Hub (WiFi-Device) to operate. 

Reviewer: ```suggestion
RF-based things need a SmartHub(WiFi-Device) to operate. 
```

Not sure if it should be Hub or SmartHub, or Smarthub, but it this moment this description and the table below don;t have a consistent naming for the hub

|           | brightness       | Dimmer                 | Brightness 0-100%                   | L510, L530, L610, L630, L900, L920                               |
|           | colorTemperature | Number                 | White-Color-Temp 2500-6500K         | L510, L530, L610, L630, L900, L920                               |
|           | color            | Color                  | Color                               | L530, L630, L900, L920                                           |
| sensor    | isOpen           | Switch                 | Contact (Door/Window) is Open       | T110                                                             |
|           | currentTemp      | Number:Temperature     | Current Temperature                 | T310, T315                                                       |
|           | currentHumidity  | Number:Dimensionless   | Current relative humidity in %      | T310, T315                                                       |

> | effects   | fxName           | String                 | Active lightning effect             | L530, L900, L920, L930                                                           |

Reviewer: Minor: can you fix the table markdown?
